api/v1/chiaseeder_types.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ChiaSeederSpec defines the desired state of ChiaSeeder
+type ChiaSeederSpec struct {
+	CommonSpec `json:",inline"`
+
+	// ChiaConfig defines the configuration options available to Chia component containers
+	ChiaConfig ChiaSeederSpecChia `json:"chia"`
+}
+
+// ChiaSeederSpecChia defines the desired state of Chia component configuration
+type ChiaSeederSpecChia struct {
+	CommonSpecChia `json:",inline"`
+
+	// BootstrapPeer a peer to bootstrap the seeder's peer database
+	// +optional
+	BootstrapPeer *string `json:"bootstrapPeer"`
+
+	// MinimumHeight only consider nodes synced at least to this height
+	// +optional
+	MinimumHeight *uint64 `json:"minimumHeight"`
+
+	// DomainName the name of the NS record for your server with a trailing period. (ex. "seeder.example.com.")
+	DomainName string `json:"domainName"`
+
+	// Nameserver the name of the A record for your server with a trailing period. (ex. "seeder-us-west-2.example.com.")
+	Nameserver string `json:"nameserver"`
+
+	// TTL ttl setting in the seeder configuration
+	// +optional
+	TTL *uint32 `json:"ttl"`
+
+	// Rname an administrator's email address with '@' replaced with '.'
+	// +optional
+	Rname *string `json:"rname"`
```

Comment:
These settings still need to be baked into chia-docker's entrypoint script. I did it on my fork, just need to move over those variables to the actual image.

Reply:
This is done in https://github.com/Chia-Network/chia-docker/pull/232